source/agora/network/NetworkManager.d

```diff
@@ -850,6 +855,19 @@ public class NetworkManager
 
             do
             {
+                auto start_height =
+                    (block_height <= 1) ?  0 : block_height;
+                auto end_height = start_height +
+                    (block_height <= 1) ?  this.consensus_config.validator_cycle :
+                        this.consensus_config.validator_cycle - 1;
```

Comment:
Why is `1` a special value here ?

Reply:
I found out that `1` is not needed if I make the formula more correct and simpler.
I changed it. Ty.